slack-api-model-kotlin-extension/src/main/kotlin/com/slack/api/model/kotlin_extension/block/composition/dsl/RichTextObjectDsl.kt

```diff
@@ -0,0 +1,75 @@
+package com.slack.api.model.kotlin_extension.block.composition.dsl
+
+import com.slack.api.model.block.element.RichTextSectionElement.LimitedTextStyle
+import com.slack.api.model.block.element.RichTextSectionElement.TextStyle
+import com.slack.api.model.kotlin_extension.block.BlockLayoutBuilder
+import com.slack.api.model.kotlin_extension.block.element.BroadcastRange
+
+@BlockLayoutBuilder
+interface RichTextObjectDsl {
+
+    /**
+     * @param range The range of the broadcast.
+     * @see <a href="https://api.slack.com/reference/block-kit/blocks#broadcast-element-type">Rich text object documentation</a>
+     */
+    fun broadcast(range: BroadcastRange)
+
+    /**
+     * @param value The hex value for the color.
+     * @see <a href="https://api.slack.com/reference/block-kit/blocks#color-element-type">Rich text object documentation</a>
+     */
+    fun color(value: String)
+
+    /**
+     * @param channelId The ID of the channel to be mentioned.
+     * @param style An object of six optional boolean properties that dictate style: bold, italic, strike, highlight, client_highlight, and unlink.
+     * @see <a href="https://api.slack.com/reference/block-kit/blocks#channel-element-type">Rich text object documentation</a>
+     */
+    fun channel(channelId: String, style: LimitedTextStyle? = null)
+
+    /**
+     * @param timestamp A Unix timestamp for the date to be displayed in seconds.
+     * @param format A template string containing curly-brace-enclosed tokens to substitute your provided timestamp.
+     * @param url URL to link the entire format string to.
+     * @param fallback Text to display in place of the date should parsing, formatting or displaying fail.
+     * @see <a href="https://api.slack.com/reference/block-kit/blocks#date-element-type">Rich text object documentation</a>
+     */
+    fun date(timestamp: Int, format: String, url: String? = null, fallback: String? = null)
```

Comment:
same as above

Reply:
Resolved in 1be897c8ce41fb2d32ee1b4cdaa0d647fd7b43c0